project-koku/koku
# [COST-1855] - add processor for Source.Update events to handle cluster-id updates for OCP sources

Testing:
1. Create an OCP source
2. Update the cluster-id for that source
3. See the updated cluster-id in cost-mgmt: `/api/cost-management/v1/sources/`
